cortx-motr.spec.in

```diff
@@ -131,7 +131,9 @@ Requires:       libyaml
 Requires:       openssl
 Requires:       gdb
 Requires:       genders
+%if %{rhel} <= 7
```

Comment:
Using closed range in comparison is easier to comprehend, consider changing to
```
%if %{rhel} < 8
```

Reply:
Agree, updated. Thanks, @chumakd!